driver/pace/driver/run.py

```diff
@@ -459,47 +549,55 @@ def __init__(
             comm: communication object behaving like mpi4py.Comm
         """
         self.config = config
-        communicator = pace.util.CubedSphereCommunicator.from_layout(
-            comm=comm, layout=self.config.layout
-        )
-        quantity_factory, stencil_factory = _setup_factories(
-            config=config, communicator=communicator
-        )
+        self.comm = comm
+        self.performance_config = self.config.performance_config
+        self.timestep_timer = self.performance_config.timestep_timer
+        self.timer = self.performance_config.total_timer
+        self.times_per_step = self.performance_config.times_per_step
+        self.hits_per_step = self.performance_config.hits_per_step
+        self.timer.start("total")
```

Comment:
how dangerous is it that this timer starts in one method and is stopped in another one?

Reply:
Could be dangerous, but we still have a null timer when performance mode is false. 

The other option is to not have total time at all, I think initialization is what we're after. The total time should be the same as initialization + all mainloop + time to write out performance output file (which we do not time), but when submitted through slurm, we should have the real total time anyways. I was trying to keep ~identical to `dynamics.py`, but we can definitely change that.